### ISSUE TITLE
Search pane doesn't appear when Windows 10 is running on High Contrast

### ISSUE THREAD

**Issue by warnov** [microsoft/terminal#4393]
# Environment

```none
Windows build number: Microsoft Windows [Version 10.0.18362.592]
Windows Terminal version (if applicable): 0.8

Any other software?
```

# Steps to reproduce

Change the Windows theme to High Contrast (black). Open Windows Terminal. Press the shortcut assigned to the `Find` command. Notice that the search pane won't appear. Changing back the theme out from High Contrast, and pressing the the shortcut assigned to the `Find` command will make the search pane appear correctly.

# Expected behavior

Changing the Windows theme to High Contrast (black) should allow to still having access to the  search pane. 

# Actual behavior
Changing the Windows theme to High Contrast (black) is not allowing to have access to the  search pane. 

When you press the key shortcut assigned to the `Find` command it is not being shown. Nothing happens.

**Comment by DHowett-MSFT** [comment]
_Wow._ Yeah, that's bad.

**Comment by warnov** [comment]
Awesome!! Thanks a lot guys for the good work and for being inclusive with the ones that need HC mode to work better.
